apps/apps.go

```diff
@@ -128,7 +133,8 @@ func Logs(c *deis.Client, appID string, lines int) (string, error) {
 	return string(body[2 : len(body)-1]), nil
 }
 
-// Run one time command in an app.
+// Run an one time command in your app. This will start a kubernetes job with the
```

Comment:
Typo with `an`, and I think one-time should be hyphenated: `Run a one-time command in your app.`.